SingularityService/src/main/java/com/hubspot/singularity/data/SandboxManager.java

```diff
@@ -91,7 +92,7 @@ public SlaveNotFoundException(Exception e) {
       }
 
       return Optional.of(objectMapper.readValue(response.getResponseBodyAsStream(), MesosFileChunkObject.class));
-    } catch (ConnectException ce) {
+    } catch (ExecutionException | ConnectException ce) {
```

Comment:
Ah gotcha, was reading it wrong as the type of exception was different, I can update that